cerberus/urls/urls.py

```diff
@@ -6,8 +6,46 @@
 
 urlpatterns = (
     [
-        path("", views.dashboard, name="dashboard"),
-        path("bookings/<int:pk>/action/<str:action>", views.BookingStateActions.as_view(), name="booking_action"),
+        path(
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (complexity):** Consider maintaining the original single-line format for simpler paths to enhance readability and manage complexity.

The refactoring introduces additional complexity and visual clutter by significantly increasing the line count and breaking down each `path` function call into multiple lines. This not only makes the `urlpatterns` list longer but also introduces a level of repetition, especially with the booking calendar views (year, month, day), that could be abstracted to reduce both repetition and complexity.

While the intention to improve readability is appreciated, the original single-line format for each path was more consistent and, for many, easier to quickly scan through. The addition of new features (calendar views and invoice actions) inherently increases complexity, but the way these new paths are interspersed with existing ones without clear separation or grouping could make the codebase harder to maintain.

Consider maintaining the original formatting for simpler paths and only using multi-line formatting for more complex paths if it genuinely improves readability. Additionally, grouping related paths or abstracting similar patterns could help manage complexity while keeping the codebase clean and maintainable.